tutorials/generate-events.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: page
+title: Generating Events
+---
+## Purpose
+
+Kite applications work with Big Data. This example class, `GenerateEvents.java`, generates 1-1.5 million random event records, a small amount of realistic Big Data you can use with Kite examples.
```

Comment:
The purpose is to fill the dataset with sample data that can be used in the following tutorials. This simulates the data that a web application (link to the Flume web app tutorial) would send if it were running for a long time. Because we don't have a long time for it to run, this is needed.

I don't think you need the part about "Big Data" because it doesn't clarify what is happening and this doesn't produce a big dataset. In the end, this introduces "Big Data" to say that this tutorial module doesn't produce big data.